.github/workflows/ci-plugin-server.yml

```diff
@@ -156,7 +156,7 @@ jobs:
         env:
             REDIS_URL: 'redis://localhost'
             CLICKHOUSE_HOST: 'localhost'
-            CLICKHOUSE_DATABASE: 'posthog_test'
+            CLICKHOUSE_DATABASE: 'posthog'
```

Comment:
I'd love if we kept posthog_test here , these tests for sure I'd love to run in a separate db and they do work there